website_docs/exporters.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Exporters
+aliases: [/docs/instrumentation/go/exporting_data]
+weight: 4
+---
+
+In order to visualize and analyze your
+[traces](/docs/concepts/signals/traces/#tracing-in-opentelemetry) and metrics, you
+will need to export them to a backend.
+
+## OTLP Exporter
+
+OpenTelemetry Protocol (OTLP) export is available in the
+`go.opentelemetry.io/otel/exporters/otlp/otlptrace` and
+`go.opentelemetry.io/otel/exporters/otlp/otlpmetric` packages.
+
+Please find more documentation on
+[GitHub](https://github.com/open-telemetry/opentelemetry-go/tree/main/exporters/otlp)
+
+## Jaeger Exporter
+
+Jaeger export is available in the `go.opentelemetry.io/otel/exporters/jaeger`
```

Comment:
How about including the content of #3705 here, to avoid losing data between the two PRs?

Reply:
would be good, since this PR is outstanding for a month now 👍